media-source/MediaSource-isTypeSupported.html

```diff
@@ -0,0 +1,45 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset='utf-8'>
+  <title>MediaSource.isTypeSupported()</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<div id="log"></div>
+<script>
+test(function() {
+    var video = document.createElement('video');
+    ['video/webm; codecs="vorbis,vp8"', 'video/mp4'].forEach(function(mime) {
+        if(MediaSource.isTypeSupported(mime)) {
+            assert_true(video.canPlayType(mime) == 'maybe' || video.canPlayType(mime) == 'probably');
```

Comment:
Not yet tested. I think another independent PR is already in the stack for adding this. If not, I'll add such a test separately.